dist/index.js

```diff
@@ -191,12 +193,12 @@ var AxisChartWidget = /** @class */ (function () {
             }
         };
     };
-    AxisChartWidget.prototype.$onInit = function () {
+    AxisChartWidget.prototype.$onChanges = function () {
```

Comment:
`$onChanges` reloads the data if the date changes.

Reply:
The dist file is built the project and shouldn't be edited manually, since it will be overwritten on every build. It's better to do these types of changes in the corresponding component files.

Reply:
As you committed the dist files, I committed my changes to them as well, so there is always a running example in the dist folder. Do you want me to revert the changes in this folder?

Reply:
the dist files are necessary for the distribution to work via npm, however they are built by the project when you run `yarn start` or similar. So they would be overwritten when the next person commits their changes.

A better place to do the edit would be the axischartwidgets file, which would then compile the changes into the distfile with the changes from everyone else.